resources/providers/azurelib/provider.go

```diff
@@ -61,16 +63,22 @@ func (p *ProviderInitializer) Init(log *logp.Logger, azureConfig auth.AzureFacto
 		return nil, fmt.Errorf("failed to init monitor client: %w", err)
 	}
 
+	genericARMClient, err := arm.NewClient("armsecurity-custom", "v0.0.1", azureConfig.Credentials, nil)
```

Comment:
What does this version stands for?

Reply:
Nothing in our case. The generic arm client asks for a module name and a version string. In our case, I put those just to not put empty strings.